src/cljs/main/broadfcui/common/notifications.cljs

```diff
@@ -199,4 +173,98 @@
+               (make-li "All use of FireCloud by researchers is subject to the "
+                        (links/create-external
+                         {:href "https://software.broadinstitute.org/firecloud/documentation/article?id=6819"}
+                         "FireCloud Terms of Use") ", which may be updated from time to time.
+                         FireCloud reserves the right to revoke credits for any activity that violates the Terms of Use.")])
+            (let [div-id (gensym "eula")]
+              [:div {:id div-id}
+               [:style {}
+                (str "#" div-id " .markdown-body strong {text-decoration: underline}\n"
+                     "#" div-id " .markdown-body ol {counter-reset: item}\n"
+                     "#" div-id " .markdown-body li:before {content: counters(item, \".\") \".\";\n"
+                                "  counter-increment: item; margin-left: -2em; position: absolute}\n"
+                     "#" div-id " .markdown-body li li:before {margin-left: -3em}\n"
+                     "#" div-id " .markdown-body li li li:before {margin-left: -4em}\n"
+                     "#" div-id " .markdown-body li {display: block}")]
+               [markdown/MarkdownView {:text eula}]
+               [:div {:style {:padding "1rem" :marginTop "0.5rem"
+                              :border style/standard-line :background (:background-light style/colors)}}
+                [:label {:style {:marginBottom "0.5rem" :display "block"}}
+                 [:input {:type "checkbox" :onChange #(swap! state update :terms-agreed? not)}]
+                 "I agree to the terms of this Agreement."]
+                [:label {:style {:display "block"}}
+                 [:input {:type "checkbox" :onChange #(swap! state update :cloud-terms-agreed? not)}]
+                 "I agree to the Google Cloud Terms of Service."]
+                [:div {:style {:paddingLeft "1rem"}} "Google Cloud Terms of Service: "
+                 (links/create-external {:href "https://cloud.google.com/terms/"}
+                                        "https://cloud.google.com/terms/")]]]))]
+         :data-test-id "eula-modal"
+         :button-bar (flex/box
+                      {:style {:justifyContent "center" :width "100%"}}
+                      (when page-2?
+                        [buttons/Button
+                         {:text "Back"
+                          :style {:marginRight "2rem"}
+                          :onClick #(swap! state dissoc :page-2?)}])
+                      [buttons/Button
+                       (if-not page-2?
+                         {:text "Review Terms of Service"
+                          :onClick #(swap! state assoc :page-2? true)}
+                         {:text "Accept"
+                          :disabled? (when-not (and terms-agreed? cloud-terms-agreed?)
+                                       "You must check the boxes to accept the agreement.")
```

Comment:
when testing locally, I could successfully click Accept without ticking the checkboxes.